pylsp/config/schema.json

```diff
@@ -370,12 +370,12 @@
       "description": "Enable or disable the plugin."
     },
     "pylsp.rope.extensionModules": {
-      "type": ["null", "string"],
+        "type": ["string", "null"],
```

Comment:
Note, I re-ordered the array values here to be consistent with the other type fields.

Reply:
The whitespace introduced should be removed in the second commit. Not sure where it came from.